package.json

```diff
@@ -4,15 +4,15 @@
   "description": "",
   "scripts": {
     "prebuild": "rm -rf node_modules/.cache && bin/buildStatic.sh",
-    "check:node": "check-node-version --node \">=8.1 <9\"",
+    "check:node": "check-node-version --package",
     "test": "npm run lint",
     "dev": "npm run prebuild && next dev",
     "build": "npm run prebuild && next build",
     "start": "next start",
     "deploy:dev": "cp deployment/development.json now.json && now deploy && now alias",
     "deploy:prod": "cp deployment/production.json now.json && now deploy && now alias",
-    "deploy:ci:dev": "npm run deploy:dev -- --token=E4RjOF6qkG8lj63zfDmeW0hT",
-    "deploy:ci:prod": "npm run deploy:prod -- --token=E4RjOF6qkG8lj63zfDmeW0hT",
+    "deploy:ci:dev": "cp deployment/development.json now.json && now deploy --token=$NOW_TOKEN --team=zdev && now alias --token=$NOW_TOKEN --team=zdev",
+    "deploy:ci:prod": "cp deployment/production.json now.json && now deploy --token=$NOW_TOKEN --team=zdev && now alias --token=$NOW_TOKEN --team=zdev",
     "postinstall": "npm run check:node",
```

Comment:
Yeah, I actually remember you doing this with the function-camp-website one. But looks good. I can't merge however.